src/components/timeline/status.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Alert from './alert';
+import Spinner from './spinner';
+import ErrorMessageContainer from './error-message.container';
+
+const Status = ( { status } ) => {
+	switch ( status ) {
+		case 'fetching':
+			return <Spinner />;
+		case 'notready':
+			// @TODO Translate
```

Comment:
Are we doing localization in another ticket?

Reply:
We should do this in https://phabricator.wikimedia.org/T181172 since the message format is going to have to change.